audiorecorder/src/main/java/org/odk/collect/audiorecorder/DaggerSetup.kt

```diff
@@ -65,7 +65,7 @@ internal open class AudioRecorderDependencyModule {
 
     @Provides
     open fun providesRecorder(application: Application): Recorder {
-        return MediaRecorderRecorder(application.cacheDir) { RealMediaRecorderWrapper(MediaRecorder()) }
+        return RecordingResourceRecorder(application.cacheDir) { MediaRecorderRecordingResource(MediaRecorder()) }
```

Comment:
Naming is hard. I'm attempting to make it clear that this could be implemented using something other than `MediaRecorder`. It's the direction I think we'd need to take if we look at a lossless codec.